py-kms/pykms_WebUI.py

```diff
@@ -0,0 +1,141 @@
+import os, uuid, datetime
+from flask import Flask, render_template
+from pykms_Sql import sql_get_all
+from pykms_DB2Dict import kmsDB2Dict
+
+def _random_uuid():
+    return str(uuid.uuid4()).replace('-', '_')
+
+_serve_count = 0
+def _increase_serve_count():
+    global _serve_count
+    _serve_count += 1
+
+def _get_serve_count():
+    return _serve_count
+
+_kms_items = None
+_kms_items_ignored = None
+def _get_kms_items_cache():
+    global _kms_items, _kms_items_ignored
+    if _kms_items is None:
+        _kms_items = {}
+        _kms_items_ignored = 0
+        queue = [kmsDB2Dict()]
+        while len(queue):
+            item = queue.pop(0)
+            if isinstance(item, list):
+                for i in item:
+                    queue.append(i)
+            elif isinstance(item, dict):
+                if 'KmsItems' in item:
+                    queue.append(item['KmsItems'])
+                elif 'SkuItems' in item:
+                    queue.append(item['SkuItems'])
+                elif 'Gvlk' in item:
+                    if len(item['Gvlk']):
+                        _kms_items[item['DisplayName']] = item['Gvlk']
+                    else:
+                        _kms_items_ignored += 1
```

Comment:
Python now supports switch-like statements, please use that.

Reply:
Hmm, this seems to be a very new feature with Python 3.10. Yes, we state that `py-kms` is tested using Python `3.10.1` (see readme), but should we really intentionally break older Python versions over this? I mean, if we are already past that point, we might as well do it, but I'll have to check that first...